index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+
+const app = express()
+const PORT = 1400
```

Comment:
The README asked you to use port 1337. You should be treating that as your documentation. I should be able to control click on the link in the documentation and be taken directly where I am trying to go to check your work. You can use any port you want with your own projects and note it in your own documentation but for assignments that is part of it. IF you opt to change it, make the README reflect your change.